components/formats-bsd/src/loci/formats/in/TileJPEGReader.java

```diff
@@ -161,4 +134,20 @@ public void initFile(String id) throws FormatException, IOException {
     MetadataTools.populatePixels(store, this);
   }
 
+  @Override
+  public void reopenFile() throws IOException {
+    if (in != null) {
+      in.close();
+    }
+    in = new RandomAccessInputStream(currentId);
+    in.seek(0);
```

Comment:
It's outwith this PR, but if opening a file for input might start with the file position anywhere other than the beginning, it might be good to add that warning to the constructor's Javadoc.